MLS.Agent/KernelExtensions.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Clockwise;
+using Microsoft.DotNet.Interactive;
+using XPlot.DotNet.Interactive.KernelExtensions;
+
+namespace MLS.Agent
+{
+    public static class KernelExtensions
+    {
+        public static T UseDefaultExtensions<T>(this T kernel)
```

Comment:
The method name doesn't convey what it's actually doing. Can we make it more specific?

Reply:
This reads pretty clearly to me as "UseXPlot"